numba/cpython/hashing.py

```diff
@@ -393,15 +370,9 @@ def impl(cgctx, builder, signature, args):
 
 @overload_method(types.BaseTuple, '__hash__')
 def tuple_hash(val):
-    if _py38_or_later or isinstance(val, types.Sequence):
-        def impl(val):
-            return _tuple_hash(val)
-        return impl
-    else:
-        def impl(val):
-            hashed = _Py_hash_t(_tuple_hash_resolve(val))
```

Comment:
Is `_tuple_hash_resolve` now unused? I would guess that it is and so it could be removed entirely?